windows/process.go

```diff
@@ -0,0 +1,30 @@
+// +build windows
+
+package windows
+
+import (
+	"github.com/containerd/containerd"
+	"github.com/containerd/containerd/windows/hcs"
+	"golang.org/x/net/context"
```

Comment:
Oh, I see, the grpc generator still uses the old one, so I have to do the same since it's passed down directly to the process object